Add Jenkins PR Trigger Guide (#399)
jfrog/frogbot: docs/templates/jenkins/README.md

[Go back to the Installation documentation page](../../../README.md)

# Setup Frogbot Using Jenkins

<div>
<img src="../../../images/jenkins-logo.png" width="250">
<img src="../../../images/frogbot-circle.png" width="180">
</div>

### 🖥️ Follow These steps to install Frogbot on Jenkins

<details>
  <summary>1️⃣  Install Jenkins Webhook Plugin </summary>

- **Install Generic Webhook Trigger**
    - Using the GUI: From your Jenkins dashboard navigate to Manage Jenkins > Manage Plugins and select the Available
      tab. Locate this plugin by searching
      for - [Generic Webhook Trigger](https://plugins.jenkins.io/generic-webhook-trigger/).

</details>
<details>

>   <summary>2️⃣ Setup Webhook on your Git Provider </summary>

Reviewer: ```suggestion
  <summary>2️⃣ Set up Webhook on your Git Provider </summary>
```

- Read more [JobToken Docs](https://plugins.jenkins.io/generic-webhook-trigger/#plugin-content-trigger-only-specific-job)
- 🌟 Choose your Git provider:
    <details>
      <summary> Bitbucket Server  </summary>

    - Go to repository settings and select Webhooks , and create a new webhook.
      <img src="../../../images/bitbucket-webhook-setup.png">
    - Set the webhook URL  `https://jenkinsUrl/generic-webhook-trigger/invoke?token=JobToken`
      <img src="../../../images/bitbucketserver-create-webhook.png">

    </details>

  <details>
      <summary> GitHub  </summary>
  
    - Go to repository settings and create a new webhook.
    <img src="../../../images/github-new-webhook.png">
  
    - Add a new webhook
    <img src="../../../images/github-webhook-setup.png">
  
    - Set up trigger
    <img src="../../../images/github-trigger-event.png">

  </details>

  <details>
        <summary> Azure Repos  </summary>

    - [Set Up Azure Repos Jenkins Webhook](https://learn.microsoft.com/en-us/azure/devops/service-hooks/services/jenkins?view=azure-devops)

    </details>

  <details>
        <summary> GitLab  </summary>

    - Go your project settings and select webhooks.

>     - Setup a webhhok with merge request events

Reviewer: ```suggestion
    - Setup a webhook with merge request events
```

>     - Setup a webhook with merge request events

Reviewer: ```suggestion
    - Set up a webhook with merge request events
```
